lib/cli/interactive-setup/dashboard-login.js

```diff
@@ -18,8 +18,11 @@ const loginOrRegisterQuestion = async (stepHistory) =>
 
 const steps = {
   loginOrRegister: async (context) => {
-    const result = await loginOrRegisterQuestion(context.stepHistory);
-    if (result) {
+    const shouldLoginOrRegister =
+      Boolean(context.options.org) ||
+      Boolean(context.configuration.org) ||
+      (await loginOrRegisterQuestion(context.stepHistory));
```

Comment:
There's no need for `Boolean` wrappers.

Sometimes it's good to ensure that given property, or function result, consistently represents boolean (it's also about not holding onto some objects which otherwise could be cleared from memory), but here it's just part of internal conditional.